packages/periphery/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@uma/periphery",
```

Comment:
just wanted to ask about the name of this package. `periphery` does make sense but `admin-scripts` or `governance-scripts` might be more descriptive? not too convinced the ones I proposed are better but just wanted to discuss it.

Reply:
I was thinking this package would include other helper scripts like `AdvanceVotePhase` and `CreatePriceRequest` which are both better described as `voting-scripts`. WDYT?

Reply:
`voting-scripts` is better imo than `periphery`. it's certainly more descriptive to what the packing contains.

Reply:
but not all of these will be voting scripts, some will be interacting with the LSP/financial contracts right? Like there might be one that creates a sponsor position. Do you think there should be a separate package for financial-contract-scripts?

I'm just imagining that every single script that interacts with any contract in `core` would go into `periphery`. And they're all closely related in that they need a similar `hardhat` setup and even a connection to a mainnet fork.

Reply:
ye see your point. how about `periphery-scripts` :)? or just `scripts`!